detection_osnet/Window  window
uint16 assignment